src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,109 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createQuery = "INSERT INTO drivers(name, license_number) VALUES(?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createDriverStatement
+                        = connection.prepareStatement(createQuery,
+                             Statement.RETURN_GENERATED_KEYS)) {
+            createDriverStatement.setString(1, driver.getName());
+            createDriverStatement.setString(2, driver.getLicenseNumber());
+            createDriverStatement.executeUpdate();
+            ResultSet generatedKeys = createDriverStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(1, Long.class);
+                driver.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create a " + driver, e);
+        }
+        return driver;
+    }
+
+    @Override
+    public Optional<Driver> get(Long id) {
+        String getQuery = "SELECT * FROM drivers WHERE id = ? AND NOT is_deleted;";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement getDriverStatement
+                        = connection.prepareStatement(getQuery)) {
+            getDriverStatement.setLong(1, id);
+            ResultSet resultSet = getDriverStatement.executeQuery();
+            if (resultSet.next()) {
+                return Optional.of(getDriver(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't get driver with id " + id, e);
+        }
+        return Optional.empty();
+    }
+
+    @Override
+    public List<Driver> getAll() {
+        String getAllQuery = "SELECT * FROM drivers WHERE NOT is_deleted";
```

Comment:
semicolon symbol ';' omitted at the end of this query string. And check also ManufacturerDaoImpl